src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -17,7 +18,8 @@ export default class VivViewerLayer extends CompositeLayer {
       sliderValues,
       colorValues,
       channelIsOn,
-      domain
+      domain,
```

Comment:
can you remind me what `domain` is for?

Reply:
This is used for determining max/min for slider values, I believe.